src/cdk-experimental/listbox/listbox.ts

```diff
@@ -383,6 +427,10 @@ export class CdkListbox<T> implements AfterContentInit, OnDestroy, OnInit, Contr
 
     if (!this.useActiveDescendant) {
       this._activeOption.focus();
+    } else {
+      if (document.activeElement === this._activeOption.getElementRef().nativeElement) {
+        this._elementRef.nativeElement.focus();
+      }
```

Comment:
I don't quite follow this- why would the focus move back to the listbox?

Reply:
This was something I added last night that might not make sense. But basically if the listbox is using aria-active descendant it was my understanding that we wouldn't want to focus the options on click. The way I had it implemented the options wouldn't focus when reached via keyboard but on click they would. This moved the focus back to the listbox.

Reply:
It's a bit of a strange way to do it but that's what I could figure out last night. If there's a better way to do it then I can remove this for now and just note it as a current bug to fix this weekend.